recipes-security/refpolicy/refpolicy-mcs-2.%/policy/modules/services/vusb.fc

```diff
@@ -21,6 +21,5 @@
 /usr/bin/usbowls        --      gen_context(system_u:object_r:vusbc_exec_t,s0)
 /usr/sbin/vusb-daemon	--	gen_context(system_u:object_r:vusbd_exec_t,s0)
 /etc/USB_fakenames.conf	-l	gen_context(system_u:object_r:vusbd_etc_t,s0)
+/etc/USB_always.conf	--	gen_context(system_u:object_r:vusbd_etc_t,s0)
```

Comment:
/etc/USB_fakenames.conf	-l	gen_context(system_u:object_r:vusbd_etc_t,s0)
/etc/USB_always.conf	--	gen_context(system_u:object_r:vusbd_etc_t,s0)

With the /config/etc substitution, this will apply to the symlinks in /etc and files in /config/etc, so "-l" for fakenames won't apply the policy properly.  If we make both "--" to label only the files, then the symlinks with be etc_t (which I think is fine) and then the /config/etc/ files are labeled.  We could also drop "--" and "-l" and let both be labeled vusbd_etc_t.  I think permissions for either are already there.

Reply:
Indeed, I did not catch the `-l`.